### ISSUE TITLE
[Abstract] Non Euclidean Graph Tutorial

### ISSUE THREAD

**Issue by meliketoy** [PyTorchKR/Tutorial-Competition-2018#8]
1. Non Euclidean Space data 중, Graph Handling 에 대한 전반적인 소개

2. 리뷰할 논문
- [Geometric Deep Learning: Going Beyond Euclidean Data](https://arxiv.org/pdf/1611.08097.pdf)
- [Revisiting Semi-Supervised Learning with Graph Embeddings](https://arxiv.org/pdf/1603.08861.pdf)
- [Semi-Supervised Classification with Graph Convolutional Networks](https://arxiv.org/pdf/1609.02907.pdf)
- [Convolutional Neural Networks on Graphs with Fast Localized Spectral Filtering](https://arxiv.org/pdf/1606.09375.pdf)
- [Graph Attention Networks](https://arxiv.org/pdf/1710.10903.pdf)
- [Distance Metric Learning using Graph Convolutional Networks: Application to Functional Brain Networks](https://arxiv.org/pdf/1703.02161.pdf)
- [MolGAN](https://arxiv.org/pdf/1805.11973.pdf)

3. Non-Euclidean Space 설명 / Graph Data 설명 / GCN / GAT /  Graph-Siamese-Network / Graph GAN 의 구현

**Comment by meliketoy** [comment]
Github 튜토리얼 링크 : https://github.com/meliketoy/graph-tutorial.pytorch
docker image 링크 : https://cloud.docker.com/repository/docker/meliketoy/pt_kr_bumsoo_kim/general
docker image run : docker run -it meliketoy/pt_kr_bumsoo_kim python [:단원]/train.py

**Comment by meliketoy** [comment]
[2018.12.27. 11:00 am] 오타 수정
